src/app/shared/dialog/dialog.component.html

```diff
@@ -1,12 +1,12 @@
-<form novalidate>
+<form novalidate aria-labelledby="modal-basic-title" aria-describedby="modal-body" role="dialog" aria-modal="true">
   <div class="modal-header">
     <span class="h4 modal-title" id="modal-basic-title">{{dialog.title | async}}</span>
     <button *ngIf="dialog.close" type="button" class="close" aria-label="Close" (click)="dialog.close.action()" [disabled]="dialog.close.disabled() | async">
       <span aria-hidden="true">&times;</span>
     </button>
   </div>
-  <div class="modal-body">
-    <scholars-alert [location]="location"></scholars-alert>
+  <div class="modal-body" id="modal-body">
```

Comment:
This `id` attribute has a chance of being non-distinct if there ar emultiple dialogs on a single page.